### ISSUE TITLE
Main stage disappearing

### ISSUE THREAD

**Issue by bgoetzmann** [in-sideFX/UndecoratorBis#8]
Hello,

I use UndecoratingBis (big thanks) in my JavaFX 8 application, and also JavaFX TextInputDialog class that is displayed with its showAndWait method in order for the user to type some text. All is OK, but no when I'm in full screen mode: when the input box is displayed the main window (stage) disappears! It reappears when the input text dialog is closed.

I'm under Windows 10.

Any idea on this problem?

Cheers,

Bertrand

**Comment by in-sideFX** [comment]
Hi Bertrand,
Unfortunately no idea. 
Is it a regression? Do you reproduce the same with a simple undecorated stage? 
Afaik, full screen mode is buggy on Mac only ;-). 
Undecorator doesn't add specific behavior in full screen mode,  so I suspect a window issue in the Glass layer. 
Thanks.

**Comment by bgoetzmann** [comment]
Thanks for your response.
I got the same problem also for JavaFX 8 Alert dialogs, so I suppose the problem concerns Dialog instances.
Finally, I found a solution that works: call initOwner on your Alert or TextInputDialog object, passing the primary stage.
To have a better looking, I also call initStyle(StageStyle.UTILITY)

A least on Windows, I'm not sure that it's a bug, because, while using fullscreen mode, perhaps the desktop window is considered as the window owner of dialogs if you don't specify an "owner".

Cheers,

Bertrand
